Xamarin.Essentials/Clipboard/Clipboard.shared.cs

```diff
@@ -6,7 +6,14 @@ namespace Xamarin.Essentials
     public static partial class Clipboard
     {
         public static Task SetTextAsync(string text)
-            => PlatformSetTextAsync(text);
+        {
+            if (text == null)
```

Comment:
should be able to do -> 

text ??= string.Empty;

Reply:
Or even `return PlatformSetTextAsync(text ?? string.Empty);`

Reply:
Thanks, it would be better this way :) Sometimes I almost forget what amazing features has C# to offer :D